src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java

```diff
@@ -100,6 +107,17 @@ static void setupSourceProjectData(@Nonnull final SphereClient sourceProjectClie
                 ResourceTypeIdsSetBuilder.of().addCategories())
             .build();
 
```

Comment:
we are missing a test case for state sync. Sorry I couldn't find anything. Even cleaning method is missing the cleaning of states.

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/133/commits/71753ce5b299547ebe98486f89dd47cb289c41f7